schema/spacewalk/upgrade/susemanager-schema-4.2.14-to-susemanager-schema-4.2.15/000-almalinux-fix-uyuni-tools-repositories.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE rhnContentSource
```

Comment:
I tested the script at an Uyuni server that had the AlmaLinux 8 client tools added. Did the replacement as expected.